1528652106 ERROR  Error: error
    at C:\Users\kron4\Desktop\Web-Final\server\src\routers\newlendform.js:21:21
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\Users\kron4\Desktop\Web-Final\server\node_modules\raw-body\index.js:224:16)
1528652157 ERROR  Error: error
    at C:\Users\kron4\Desktop\Web-Final\server\src\routers\newlendform.js:21:21
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\Users\kron4\Desktop\Web-Final\server\node_modules\raw-body\index.js:224:16)
1528652451 ERROR  Error: error
    at C:\Users\kron4\Desktop\Web-Final\server\src\routers\newlendform.js:21:21
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\Users\kron4\Desktop\Web-Final\server\node_modules\raw-body\index.js:224:16)
1528655246 ERROR  TypeError: newlendModel.create is not a function
    at C:\Users\kron4\Desktop\Web-Final\server\src\routers\newlendform.js:26:18
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\Users\kron4\Desktop\Web-Final\server\node_modules\raw-body\index.js:224:16)
1528655321 ERROR  TypeError: newlendModel.create is not a function
    at C:\Users\kron4\Desktop\Web-Final\server\src\routers\newlendform.js:26:18
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\Users\kron4\Desktop\Web-Final\server\node_modules\raw-body\index.js:224:16)
1528655424 ERROR  ReferenceError: mmoney is not defined
    at Promise (C:\Users\kron4\Desktop\Web-Final\server\src\model\newlendform.js:25:20)
    at new Promise (<anonymous>)
    at Object.create (C:\Users\kron4\Desktop\Web-Final\server\src\model\newlendform.js:21:12)
    at C:\Users\kron4\Desktop\Web-Final\server\src\routers\newlendform.js:26:18
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\kron4\Desktop\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
1529331719 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529331810 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529331813 ERROR  error: database "FinalDB" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529332027 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529332040 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529332045 ERROR  error: database "FinalDB" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529332430 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529332434 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529332847 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529332849 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529332971 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529332995 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529333069 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529333077 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529333132 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529333144 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529333158 ERROR  TypeError: alertModel.list is not a function
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:13:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529333209 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333262 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333593 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333750 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333783 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333785 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333814 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333819 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333888 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333891 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333971 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529333971 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529334044 ERROR  Error: getaddrinfo ENOTFOUND undefined undefined:5432
    at errnoException (dns.js:50:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:92:26)
1529334047 ERROR  Error: getaddrinfo ENOTFOUND undefined undefined:5432
    at errnoException (dns.js:50:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:92:26)
1529334311 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529334396 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529334543 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529334549 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341037 ERROR  TypeError: req.set is not a function
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:10:9)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:317:13)
1529341037 ERROR  TypeError: req.set is not a function
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:10:9)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:317:13)
1529341042 ERROR  TypeError: req.set is not a function
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:10:9)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:317:13)
1529341042 ERROR  TypeError: req.set is not a function
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:10:9)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:317:13)
1529341053 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341056 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341060 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341067 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341333 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341555 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341555 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341555 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341555 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341555 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341555 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341555 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341555 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341555 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341555 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341960 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529341987 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342005 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342037 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342063 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342083 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342095 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342102 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342105 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342117 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342228 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342272 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342342 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342442 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342461 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342468 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342523 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529342527 ERROR  Error: Connection terminated
    at Connection.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\client.js:183:17)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Connection.emit (events.js:208:7)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at TCP._handle.close [as _onclose] (net.js:567:12)
1529512977 ERROR  Error: getaddrinfo ENOTFOUND undefined undefined:5432
    at errnoException (dns.js:50:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:92:26)
1529513437 ERROR  error: relation "alerts" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529514660 ERROR  error: relation "alerts" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529515691 ERROR  ReferenceError: name is not defined
    at Object.create (C:\Users\user\Desktop\Web\Web-Final\server\src\model\alertList.js:11:25)
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:30:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
1529516358 ERROR  ReferenceError: name is not defined
    at Object.create (C:\Users\user\Desktop\Web\Web-Final\server\src\model\alertList.js:11:25)
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:30:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
1529516458 ERROR  error: syntax error at or near "{"
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529516660 ERROR  ReferenceError: sql is not defined
    at Object.create (C:\Users\user\Desktop\Web\Web-Final\server\src\model\alertList.js:19:20)
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:30:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
1529516677 ERROR  ReferenceError: sql is not defined
    at Object.create (C:\Users\user\Desktop\Web\Web-Final\server\src\model\alertList.js:19:20)
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:30:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
1529516756 ERROR  ReferenceError: sql is not defined
    at Object.create (C:\Users\user\Desktop\Web\Web-Final\server\src\model\alertList.js:19:20)
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:42:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
1529517033 ERROR  ReferenceError: sql is not defined
    at Object.create (C:\Users\user\Desktop\Web\Web-Final\server\src\model\alertList.js:20:20)
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:30:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
1529517058 ERROR  TypeError: Empty or undefined query.
    at Database.$query (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg-promise\lib\query.js:47:17)
    at Database.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg-promise\lib\query.js:231:23)
    at config.$npm.connect.pool.then.db (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg-promise\lib\database.js:242:42)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
1529517152 ERROR  error: relation "alerts" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529517326 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529519075 ERROR  ReferenceError: lender is not defined
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:30:23
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529519146 ERROR  ReferenceError: expect_date is not defined
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:32:39
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529519257 ERROR  error: syntax error at or near "{"
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529519320 ERROR  error: syntax error at or near "{"
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529520240 ERROR  error: syntax error at or near "{"
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529521374 ERROR  ReferenceError: lender is not defined
    at Object.create (C:\Users\user\Desktop\Web\Web-Final\server\src\model\alertList.js:21:24)
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:35:16
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
1529522376 ERROR  error: column "蔡旻翰" of relation "alerts" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522450 ERROR  error: column "蔡旻翰" of relation "alerts" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522502 ERROR  error: column "蔡旻翰" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522513 ERROR  error: column "蔡旻翰" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522525 ERROR  error: column "蔡旻翰" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522748 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522762 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522802 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522847 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522901 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522919 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529522988 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529523027 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529523268 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529523273 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529523286 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529523346 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529523371 ERROR  error: column "date" does not exist
    at Connection.parseE (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
1529565236 ERROR  Error: error
    at C:\Users\user\Desktop\Web\Web-Final\server\src\routers\alertList.js:27:21
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\index.js:275:10)
    at module.exports (C:\Users\user\Desktop\Web\Web-Final\server\src\middleware\access-controller.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\Web\Web-Final\server\node_modules\express\lib\router\layer.js:95:5)
1529565236 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
1529565258 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
1529565258 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
1529565260 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
1529565273 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
1529565279 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
1529565324 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
1529565383 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
1529565383 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
1529565499 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
1529565504 ERROR  Error: connect ETIMEDOUT 54.218.231.227:5432
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
